From c458b6eb237f99aa25573cb0a106bec46141fca8 Mon Sep 17 00:00:00 2001
From: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Date: Mon, 23 Sep 2019 09:49:58 +0800
Subject: [PATCH 125/245] felix: TSN: add TSN device init function

commit 7b08959f8db7e6736d27cf9e6e4744a6e34a7050 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Add TSN device init function to init pcpmap and rtag parse set.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/mscc/felix_board.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/mscc/felix_board.c b/drivers/net/ethernet/mscc/felix_board.c
index d7b7644dce90..288d9bc1ff12 100644
--- a/drivers/net/ethernet/mscc/felix_board.c
+++ b/drivers/net/ethernet/mscc/felix_board.c
@@ -38,6 +38,7 @@ MODULE_DEVICE_TABLE(pci, felix_ids);
 
 #ifdef CONFIG_MSCC_FELIX_SWITCH_TSN
 const struct tsn_ops switch_tsn_ops = {
+	.device_init                    = felix_tsn_init,
 	.qbv_set			= felix_qbv_set,
 	.qbv_get			= felix_qbv_get,
 	.qbv_get_status			= felix_qbv_get_status,
-- 
2.17.1

